### DISCUSSION TITLE
Recently Used list ?

### DISCUSSION THREAD

**Discussion by johne53** [apohl79/audiogridder#479] [Q&A]
When I open an Audiogridder plugin (initially) I see a '+' sign at the LHS.

![Capture-01](https://user-images.githubusercontent.com/4126346/114146775-e6c28900-990f-11eb-9827-51ba7e161387.PNG)

Clicking '+' shows me a list of my VST plugins as well as a separate list of VST3's... and just above those lists there's a list of seemingly random plugins which (I'm guessing) is some kinda Recently Used list.

I can't find a way to delete entries from that Recently Used list. Is it possible?

**Comment by CaBuffalo** [comment]
There are two categories in the list, VST and VST3. On top of that is the
plugins you used recently. It shouldn't be a random list.

On Fri, Apr 9, 2021 at 12:52 AM johne53 ***@***.***> wrote:

> When I open an Audiogridder plugin (initially) I see a '+' sign at the LHS.
>
> [image: Capture-01]
> <https://user-images.githubusercontent.com/4126346/114146775-e6c28900-990f-11eb-9827-51ba7e161387.PNG>
>
> Clicking '+' shows me a list of my VST plugins as well as a separate list
> of VST3's... and just above those lists there's a list of seemingly random
> plugins which (I'm guessing) is some kinda Recently Used list.
>
> I can't find a way to delete entries from that Recently Used list. Is it
> possible?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/479>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6G7CC4HFV3LGB56YILTH2W2HANCNFSM42UOQXMA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by johne53** [reply]
Either way... can I manually remove things from those recent items?

**Comment by CaBuffalo** [comment]
No, I don't think so.

On Fri, Apr 9, 2021 at 8:10 AM johne53 ***@***.***> wrote:

> Either way... can I manually remove things from those recent items?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/479#discussioncomment-590193>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EZGZ6EXFA5SU3SQ33TH4KIFANCNFSM42UOQXMA>
> .
>


-- 
Kevin Ma
Supported by Gmail.
